README.md

```diff
@@ -26,7 +26,6 @@ Make sure [poetry][poetry] is installed
 git clone https://github.com/Clinical-Genomics/schug
 cd schug
 poetry install
-schug setup --demo
```

Comment:
This command is not working at the moment (see #34) so no point in including it in README right now, I think